openpype/tools/project_manager/project_manager/widgets.py

```diff
@@ -141,16 +144,25 @@ def __init__(self, parent=None, dbcon=None):
         inputs_widget = QtWidgets.QWidget(self)
         project_name_input = QtWidgets.QLineEdit(inputs_widget)
         project_code_input = QtWidgets.QLineEdit(inputs_widget)
+        project_width_input = QtWidgets.QLineEdit(inputs_widget)
```

Comment:
shouldn't this be Int value input

Reply:
In synchronicity this is duplicate comment ;) https://github.com/ynput/OpenPype/pull/5075#discussion_r1214146492